jerry-core/vm/vm.c

```diff
@@ -2539,6 +2539,17 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
                   || JERRY_CONTEXT (debugger_stop_context) == JERRY_CONTEXT (vm_top_context_p)))
           {
             jerry_debugger_breakpoint_hit (JERRY_DEBUGGER_BREAKPOINT_HIT);
+            if (JERRY_CONTEXT (debugger_flags) & JERRY_DEBUGGER_THROW_ERROR_FLAG)
+            {
+              result = JERRY_CONTEXT (error_value);
+              ecma_value_t to_string_value = ecma_op_to_string (result);
+              ecma_string_t *string_p = ecma_get_string_from_value (to_string_value);
+              ECMA_STRING_TO_UTF8_STRING (string_p, buffer_p, buffer_size);
+              jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Uncaught exception: %.*s\n", buffer_size, buffer_p);
+              ECMA_FINALIZE_UTF8_STRING (buffer_p, buffer_size);
```

Comment:
Do we need to log this? The user sent the request after all.

Reply:
I've removed the logging part.